app/helpers/ems_cluster_helper/textual_summary.rb

```diff
@@ -254,6 +254,16 @@ def textual_aggregate_disk_capacity
     {:label => "Total Disk Capacity", :value => number_to_human_size(@record.aggregate_disk_capacity.gigabytes, :precision => 2)}
   end
 
+  def textual_cinder_disk_usage
+    return nil unless @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::EmsCluster) && @record.name.include?("BlockStorage")
```

Comment:
@rwsu yes please, do block_storage? and object_storage? methods on cluster, while we are here we will nee also compute? and maybe controller?

Define the methods only on OpenstackInfra specific cluster, then the .respond_to?(:block_storage?) check will handle clusters without the method

We already have a usecase where ceph storage cluster has both block and object storage on it